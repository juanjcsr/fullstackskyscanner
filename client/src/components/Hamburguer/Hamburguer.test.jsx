import React from 'react';
import TestRenderer from 'react-test-renderer';

import Hamburger from './Hamburguer';

describe('Hamburguer', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<Hamburger />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
