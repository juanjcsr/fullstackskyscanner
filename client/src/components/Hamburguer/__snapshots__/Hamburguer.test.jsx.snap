// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburguer should render correctly 1`] = `
<div>
  <div
    className="hamburguer"
  />
  <div
    className="hamburguer"
  />
  <div
    className="hamburguer"
  />
</div>
`;
